takwimu/templates/profile/profile_detail_takwimu.html

```diff
@@ -316,17 +314,138 @@ <h1>Budget</h1>
       <div class="section-container">
         {% if not budget.government_expenditure_dist.is_missing %}
           <section class="clearfix stat-row">
-            <h2 class="header-for-columns">Government Spending</h2>
             <div class="column-full"
                  id="chart-grouped_column-budget-government_expenditure_dist"
-                 data-stat-type="scaled-percentage">
+                 data-stat-type="scaled-percentage" data-chart-title="Government Spending">
             </div>
           </section>
         {% endif %}
       </div>
     </article>
   {% endif %}
 
+  {% if not gender.is_missing %}
+    <article class="clearfix">
+      <header class="section-contents">
+        <h1>Gender</h1>
+      </header>
+      <div class="section-container">
+        {% if not gender.hypertension_and_diabetes.is_missing %}
+          <section class="clearfix stat-row">
+            <div class="column-full"
+                 id="chart-grouped_column-gender-hypertension_or_diabetes_sex"
+                 data-stat-type="scaled-percentage"
+                 data-chart-title="Occurence of Hypertension or Diabetes by sex">
+            </div>
+            <div class="column-full"
+                 id="chart-grouped_column-gender-hypertension_or_diabetes_agegroup"
+                 data-stat-type="scaled-percentage" data-chart-title="Occurence of Hypertension and Diabetes by age group">
+            </div>
+          </section>{% endif %}
+
+        <section class="clearfix stat-row">
+          {% if not gender.cervical_cancer.is_missing %}
```

Comment:
Move ` <section class="clearfix stat-row" ...` inside `{% if not gender....` so that if data is missing, we don't end up with empty sectoins (I think `stat-row` has borders at the top and bottom)